server/routes/createGoal/createGoalController.test.ts

```diff
@@ -83,6 +85,7 @@ describe('createGoalController', () => {
         'targetDate-year': '2024',
         action: 'add-another-step',
       }
+      req.session.addStepForms = []
```

Comment:
As above re: the expected state/content of the `addSessionsForms` array in session